src/abi/IbbtcVaultZap.abi

```diff
@@ -1,10 +1,4 @@
 [
-	{
-		"anonymous": false,
```

Comment:
I would leave the abi without changes even if we don't need functions it should match the contract